src/lib/skylark/objects.ts

```diff
@@ -177,9 +189,10 @@ export const getObjectOperations = (
   const { relationships, ...createMeta } = getMutationInfo(
     createMutation,
     objectType,
-    "create",
   );
-  const updateMeta = getMutationInfo(updateMutation, objectType, "update");
+  const updateMeta = getMutationInfo(updateMutation, objectType);
+
+  const hasRelationships = relationships.length > 0;
```

Comment:
not sure if this is enough 🤔 if not, it'll be fixed on merge

Reply:
enough to make sure that has relationships 😄

Reply:
Niceee